binding.gyp

```diff
@@ -90,17 +87,11 @@
       'include'
     ],
     'defines': [
-      'GPR_BACKWARDS_COMPATIBILITY_MODE'
+      'GPR_BACKWARDS_COMPATIBILITY_MODE',
+      'GRPC_ARES=0',
+      'GRPC_UV'
```

Comment:
No, `GRPC_UV` changes which TCP implementation is used. The `#ifdef`s in the extension code just piggyback on that.